### ISSUE TITLE
Issue in set up

### ISSUE THREAD

**Issue by AleenaAzam** [dattran2346/xray-thesis#1]
I am trying to reproduce this model you have implemented. However, being a fairly novice user, I am having trouble doing so.
Can you provide more details as to how to run and build your code (on Jupyter notebook)?

**Comment by dattran2346** [comment]
Hi @AleenaAzam,
First, you need to setup the enviroment: [pytorch](https://pytorch.org/) v1.0, [pretrained-models](https://github.com/Cadene/pretrained-models.pytorch) and fastai v0.4 (I have included a snapshot of their code in `/src/fastai`), torchvision, numpy, Pillow, scikit-image,  pandas, opencv, 

Second set up the training data, take a look at [dataset.py](https://github.com/dattran2346/chestX-ray-14/blob/master/src/dataset.py) for classification and segmentation dataset. You can download the dataset follow this [link](https://github.com/dattran2346/chestX-ray-14#dataset)

Third setup the training code by copying the model in [chexnet.py](https://github.com/dattran2346/chestX-ray-14/blob/master/src/chexnet.py) and [unet](https://github.com/dattran2346/chestX-ray-14/blob/master/src/unet.py). The notebook [Train Local.ipynb](https://github.com/dattran2346/chestX-ray-14/blob/master/src/Train%20Local%20.ipynb)  is for training classification and the notebook [Lung Segmentation.ipynb](https://github.com/dattran2346/chestX-ray-14/blob/master/src/Lung%20Segmentation.ipynb) is for training segmentation. 

To generate heatmap using CAM, take a look at [heatmap.py](https://github.com/dattran2346/chestX-ray-14/blob/master/src/heatmap.py)

For the whole inference process, you can take a look at [server.py](https://github.com/dattran2346/chestX-ray-14/blob/master/src/server.py)

**Comment by AleenaAzam** [comment]
Hi @dattran2346,
Can you please clarify a few questions:
1. What does the cut_all.pickle file contain?
2. For the training of the Unet model, the labels for the data is the binary image containing a mask of both lungs right?
3. Can you please guide us a bit more as to how to test your Unet model on another data set?
